docs/platform/concepts/free-trial.md

```diff
@@ -2,43 +2,30 @@
 title: 30-day trials
 ---
 
-Aiven offers a free trial for 30 days to explore the platform -
-including the [Aiven Console](https://console.aiven.io/),
-[API](/docs/tools/api),
-[CLI](/docs/tools/cli),
-[Aiven Terraform provider](/docs/tools/terraform), and
-[Aiven Operator for Kubernetes](/docs/tools/kubernetes) - and the [Aiven services](https://docs.aiven.io/). You
-don\'t need a credit card to sign up.
-
-If you create services on the
-[free plan](/docs/platform/concepts/free-plan), they will continue to run alongside your free 30-day trial
-without affecting your trial credits. Free plan services also continue
-running after the trial has expired.
+Aiven offers a free trial for 30 days to explore the platform - including the [Aiven Console](/docs/tools/aiven-console), [API](/docs/tools/api), [CLI](/docs/tools/cli), [Aiven Terraform provider](/docs/tools/terraform), and [Aiven Operator for Kubernetes](/docs/tools/kubernetes) - and the [Aiven services](/docs/products/services). You don't need a credit card to sign up.
 
 ## Trial features and limitations
 
-The trial credits can be used for any paid services and plan sizes as
-well functionality like virtual private cloud peering.
+Trial credits can be used for any paid services and plan sizes in any project. They can
+also be used for paid features like virtual private cloud peering.
 
 Trials include:
 
 -   Up to 10 VMs
--   1
-    [Virtual Private Cloud (VPC)](/docs/platform/howto/manage-vpc-peering)
+-   1 [Virtual Private Cloud (VPC)](/docs/platform/howto/manage-vpc-peering)
 -   Up to 10 VPC peering connections
 
-There are some limitations:
+If you create services on the [free plan](/docs/platform/concepts/free-plan), they will
+continue to run alongside your free 30-day trial without affecting your trial credits.
+Free plan services also continue running after the trial has expired.
 
--   You can only have one trial
--   Trials can only be started in your default project, which is created
-    when you sign up
--   Trial credits cannot be transferred to another project
--   You can\'t create new services if your remaining credits would be
-    spent too quickly
+There are some limitations:
 
-## Add a payment card
+-   You can only have one trial.
+-   Trial credits are added to the default
```

Comment:
This is something that has changed actually a bit but I'm not sure how important the changes are. Technically we add the trial credits to the billing group where the user performs the trial starting action (like creating a paid service). This isn't necessarily the default billing group but I would assume that in 95% of the trials it will be because people aren't creating billing groups for fun.

So not sure if this needs to be changed at all but we could just maybe say something like "Trial credits are assigned to a single billing group".

Reply:
Good to know, thanks! I've updated that part as suggested.